inventory/environ.py

```diff
@@ -411,6 +412,26 @@ def getHEC(vars_scope):
     else:
         vars_scope["splunk"]["hec"]["ssl"] = bool(vars_scope["splunk"]["hec"].get("ssl"))
 
+def getDSP(vars_scope):
```

Comment:
Annotating my own code here, mostly for reminder purposes: add unit tests for this.